src/util/KVStore/AMOCommand.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Jack Khuu, Justin Johnson, Lukas Joswiak
+
+#include "AMOCommand.hpp"
+
+#include <string>
```

Comment:
Move this include above the AMOCommand include.

Reply:
Always include your own header first -Google